config/config.exs

```diff
@@ -43,8 +43,8 @@ config :opentelemetry,
 # You can also supply opentelemetry resources using environment variables, eg.:
 # OTEL_RESOURCE_ATTRIBUTES=service.name=name,service.namespace=namespace
 
-# config :opentelemetry, :resource,
-#   service: [
-#     name: "service-name",
-#     namespace: "service-namespace"
-# ]
+config :opentelemetry, :resource,
+  service: [
+    name: "service-name",
```

Comment:
would be good to now what `resource`, `service` and `namespace` all refer to here. `service-name` and `service-namespace` look like placeholder values

Reply:
Agree. I have done some searching on that, but to no avail as yet.

Reply:
Let's try without these (commented out again), but with dummy data?

Reply:
worth a try 👍